src/DotNetLightning.Core/Serialization/LightningStream.fs

```diff
@@ -325,20 +325,22 @@ type LightningReaderStream(inner: Stream) =
         let len = this.ReadUInt16(false)
         this.ReadBytes(int32 len)
 
+    member this.ReadKey() =
+        let b: array<byte> = this.ReadBytes(Key.BytesLength)
```

Comment:
nit: no need for parens here, and now that you're gonna change this line, rename `b` to have a better name please

Reply:
Fixed.